src/py42/clients/audit_logs.py

```diff
@@ -0,0 +1,41 @@
+class AuditLogsClient(object):
+    def __init__(self, audit_log_service):
+        self._audit_log_service = audit_log_service
+
+    def get_all(
```

Comment:
Can we add a method for getting the page here?

Reply:
Done.